draft-ietf-quic-recovery.md

```diff
@@ -1147,7 +1153,10 @@ OnAckReceived(ack, pn_space):
     OnPacketsLost(lost_packets)
   OnPacketsAcked(newly_acked_packets)
 
-  pto_count = 0
+  // Reset pto_count unless the server has not yet
+  // validated the client's address.
+  if (PeerCompletedAddressValidation()):
+    pto_count = 0
   SetLossDetectionTimer()
```

Comment:
Should this also be done only when pto_count is set to 0? Why restart the timer here if the pto_count hasn't changed?

Reply:
A new RTT measurement is available, so the alarm value should always be updated.